.cicd/helpers/execute_test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+function execute-psql-command(){
+  POSTGRES_CONTAINER_ID=`docker-compose ps -q postgres`
+  CMD="docker exec -it $POSTGRES_CONTAINER_ID psql $1 -c '$2'"
+  RET_SQL_CMD=`eval $CMD`
+}
+
+if [[ "$BUILDKITE" == 'true' ]]; then
+    apt-get update && apt-get -y install wget
+fi
```

Comment:
Remove the installation of `wget` and use `curl` instead

Reply:
Done